package.json

```diff
@@ -10,12 +10,13 @@
   "scripts": {
     "build": "webpack -d --watch",
     "start": "nodemon server/index.js -w server",
```

Comment:
Your `npm start` command should generally run `node`, not `nodemon`, since your `npm start` is usually what's run in production (and `nodemon` is not necessary in production).